android/sdl_android/src/androidTest/java/com/smartdevicelink/managers/screen/menu/MenuCellTests.java

```diff
@@ -75,6 +75,7 @@ public void testSettersAndGetters() {
         menuCell.setSecondaryText(TestValues.GENERAL_STRING);
         menuCell.setTertiaryText(TestValues.GENERAL_STRING);
         menuCell.setSecondaryArtwork(TestValues.GENERAL_ARTWORK);
+        menuCell.setUniqueTitle(TestValues.GENERAL_STRING);
```

Comment:
There are some unused imports in this class, not of your doing, but let's clean them up